analyzer/consensus/consensus.go

```diff
@@ -823,7 +819,7 @@ func (m *Main) queueValidatorUpdates(batch *storage.QueryBatch, data *storage.Sc
 	for _, validator := range data.Validators {
 		batch.Queue(validatorNodeUpdateQuery,
 			validator.ID.String(),
-			validator.VotingPower,
+			fmt.Sprintf("%d", validator.VotingPower),
```

Comment:
what type was VotingPower?

Reply:
VotingPower is `int64` in go. I thought it was `uint64`. I think we can safely assume it's non-negative ... So I'm switching it in the db to `UINT31` (from `UINT_NUMERIC`), and removing the stringification here. Thanks!